### ISSUE TITLE
Reconsider/remove box-shadow floating style

### ISSUE THREAD

**Issue by kaylachumley** [patternfly/patternfly-design-kit#772]
all components that use the box-shadow floating style need to be replaced with the box-shadow medium 

delete/remove floating shadow from design tokens and styles library

update components that use that(do a once over for others):

- [x] tooltip
- [x] popover 
- [x] menu
- [x] search input panel

**Comment by kaylachumley** [comment]
Revisiting this since its been some time! @lboehling 

Looks like the tooltip on org is using global--box--shadow--medium and the tooltip in figma is using the global--box--shadow--floating. Same goes for the popover, menu and search input panel. 

Am I good to swap those components in figma to match the medium box shadow being used on org? I'll do a once over in figma to see if any other components are using that floating style and match it to org. 
I recall since the floating isn't being used at all that you were inclined to remove that token as well?

**Comment by lboehling** [comment]
Yeah, let's align with what's in core and make sure the Figma styles are consistent. Almost all (if not all) instances of shadows in our library are technically "floating," so I think it's OK to remove that token for now since a) it's not used in components, and b) its semantic use is probably too broad for the actual applications needed. We can reconsider if we find the need for better semantic shadow tokens if we add decorative shadows that aren't used on overlay/floating elements. ty @kaylachumley!

**Comment by kaylachumley** [comment]
follow up issue: https://github.com/patternfly/design-tokens/issues/99

**Comment by kaylachumley** [comment]
second exploratory follow up issue: https://github.com/patternfly/design-tokens/issues/100

**Comment by kaylachumley** [comment]
removing style from figma and created another follow up issue in token repo
